front/src/app/scenes/transaction/transaction.component.html

```diff
@@ -74,20 +74,49 @@
               <button class="btn btn-primary" [ngClass]="{'active': showUtf8}" (click)="showUtf8 = true">UTF-8</button>
             </div>
             <textarea
-                class="form-control code"
+                class="form-control code mt-2"
                 id="input_data"
                 readonly
                 rows="5"
             >{{tx.input_data | hex2str: showUtf8}}</textarea>
           </div>
           <div class="form-group">
             <label for="receipt_logs">Receipt Logs</label>
-            <textarea
-                class="form-control code"
-                id="receipt_logs"
-                readonly
-                rows="5"
-            >{{tx.logs}}</textarea>
+            <div class="btn-group btn-group-sm float-right">
+              <button class="btn btn-primary" [ngClass]="{'active': !showLogsRaw}" (click)="showLogsRaw = false">Parsed</button>
+              <button class="btn btn-primary" [ngClass]="{'active': showLogsRaw}" (click)="showLogsRaw = true">JSON</button>
+            </div>
+            <div class="mt-2">
+              <ng-container *ngIf="!showLogsRaw; else rawLogBlock;">
+                <div class="mb-2">
+                  <span id="red-box"></span> - removed log
+                </div>
+                <table class="table table-bordered">
+                  <thead>
+                  <tr>
+                    <th>Index</th>
+                    <th>Contract</th>
+                    <th>Log Data</th>
+                  </tr>
+                  </thead>
+                  <tbody>
+                  <tr *ngFor="let log of tx.processedLogs" [ngClass]="{'table-secondary': log.removed}">
+                    <td>{{log.index}}</td>
+                    <td>{{log.contract_address}}</td>
```

Comment:
This can be a link. Also, IMO using a fixed width font looks good for hex addresses.

Reply:
Updated to link.